src/cmclient/cmclient.cpp

```diff
@@ -316,21 +368,24 @@ aos::Error CMClient::SendPBMessageToVChan()
 
     err = SendBufferToVChan((uint8_t*)&header, sizeof(VChanMessageHeader));
     if (!err.IsNone()) {
+        NotifyWriteFail();
```

Comment:
You can send it once in `SendBufferToVChan`

Reply:
Done